lib/vagrant-service-manager/services/docker.rb

```diff
@@ -56,6 +58,10 @@ def self.info(machine, ui, options = {})
       end
 
       def self.print_env_info(ui, options)
+        if PluginUtil.debug_mode?
+          PluginUtil.log("script_readable: #{options[:script_readable] || false}", 'STDOUT')
```

Comment:
@LalatenduMohanty Working on fixing `one method to add debug logs`. Hope it would be clear then.